### PR TITLE
☎️ Organize the customer support form

### PR DESCRIPTION
**Pull request by thecristen**
#### Summary of changes
**Asana Ticket:** [☎️ Organize the customer support form](https://app.asana.com/0/385363666817452/1188535657153272/f)

Implements the intermediate customer support form design that will reign until we complete the epic.

* Adds reworded section headings
* Restructures the grey sections to match the mockup
* Arranges the category radio buttons vertically
* Moves and rephrases the "responses may take up to 5 days disclaimer"

There's a fair amount of code cleanup since I wanted to leave things in good shape for us to start adding new fields! Mostly I'm trying to reduce some duplication in the templates by moving a few key pieces over to functions in the Elixir view we can use to generate error messages, placeholder text, and help text. Tried to do all of this carefully as to not break any of the existing tests. :pray:

Merging into master since this change can go live before other changes to the customer support page.

---

Before getting review, please check the following:

* [x ] Does frontend functionality render and work correctly in IE?
* [x] Have we load-tested any new pages or internal API endpoints that will receive significant traffic?
* [x] Are interactive elements accessible to screen readers?
* [x] Have you checked for tech debt you can address in the area you're working in?
* [x] If this change involves routes, does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?
* [x] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

File: apps/site/assets/js/support-form.js [hunk 1]
```diff
@@ -209,12 +209,8 @@ function removeClass(node, className) {
...
 
 export function setupRequestResponse($) {
-  $("#request_response").click(function() {
-    if ($(this).is(":checked")) {
-      showExpandedForm($);
-    } else {
-      hideExpandedForm($);
-    }
+  $("#request_response").change(function() {
+    $("#contactInfoForm").toggle($(this).is(":checked"));
```
Comment (alicia-mbta) [inline apps/site/assets/js/support-form.js:213]: Nice! 👍

**Review by alicia-mbta — APPROVED** [review]
Looks great on desktop and mobile. Really cool refactors too. 💯

**Comment by ingridpierre** [comment]
@thecristen I'm not seeing the captcha anymore, is it still included in the form?

**Comment by alicia-mbta** [comment]
I see it:
image
I had to hard-refresh the browser to see the changes if it helps.

**Comment by alicia-mbta** [comment]
Also, are you using the feature branch? I think we had the 4th category ("Compliment") already available in there.

**Comment by alicia-mbta** [comment]
Disregard my latest comment if this piece of work was meant to be already available in master and not as part of the new customer support form...

**Comment by thecristen** [comment]
Forgot to mention the captcha requires a few environment variables -- I forwarded some to Ingrid. :)

**Comment by ingridpierre** [comment]
👍 lgtm now!

**Pull request merged**